articles/libraries/lock/v10/migration-guide.md

```diff
@@ -4,39 +4,74 @@ Heads up! This document is using the latest version of Lock (version 10). See ch
 
 # Lock 9 to Lock 10 Migration Guide
 
-The following instructions assume you are migrating from Lock 9 to the latest Lock 10. If you are upgrading from a preview release, please refer to the [preview changes](#preview-changes). If you came here looking for the migration guide from the "Auth0Widget" to Lock v9, here's the [legacy-migration-guide](/libraries/lock/v9/migration-guide). Otherwise, read on!
+The following instructions assume you are migrating from **Lock 9** to the latest **Lock 10**. If you are upgrading from a preview release of Lock 10, please refer to the [preview changes](#preview-changes). Otherwise, read on!

```

Comment:
@jeffreylees The [URL](https://auth0-docs-content-pr-2008.herokuapp.com/docs/libraries/lock/v10/migration-guide#preview-changes) does not work. Maybe you meant `#upgrading-from-preview-releases`?